tests/element_test_vectors.rs

```diff
@@ -390,26 +390,51 @@ mod impl_display_for_element_tests {
     use super::*;
     use ion_rs::value::native_writer::NativeElementWriter;
     use ion_rs::value::reader::element_reader;
-    use ion_rs::{IonResult, TextWriterBuilder};
+    use ion_rs::TextWriterBuilder;
     use std::fs::read;
 
+    const TO_STRING_SKIP_LIST: [&'static str; 12] = [
```

Comment:
You can get around having to state the length of the array (and updating it with each change) by using a reference:

```suggestion
    const TO_STRING_SKIP_LIST: &[&'static str] = &[
```